lib/src/parser/models/flavors/android.dart

```diff
@@ -45,6 +45,9 @@ class Android extends OS {
   @JsonKey(disallowNullValue: true)
   final AGConnect? agconnect;
 
+  @JsonKey(disallowNullValue: true)
+  final Map<String, dynamic>? adaptiveIcon;
```

Comment:
```suggestion
  final AdaptiveIcon? adaptiveIcon;
```

Sorry I haven't been clear enough: If we create a nested model we have more control over the fields.